### DISCUSSION TITLE
Pressure/Depth estimation

### DISCUSSION THREAD

**Discussion by Martina-Scanu** [dankelley/oce#2145] [Q&A] [answered]
Dear Prof. Kelley,

in order to avoid buying a new instrument, I was wondering if it is possible to derive water pressure (or depth) having salinity, temperature, and conductivity. I do not need a very precise value, but I would like to be able to distinguish at least between meters of depth. 
I already had a look at the main oce functions and I think it is not possible, but better to try! 
All my available variables are: Temp. | pH | mV | ORP | EC | EC Abs. | RES | TDS | Sal. | Sigma T | Atm.Press. | D.O. | Turb. | Coord.

Thank you in advance,
Martina

**Comment by richardsc** [comment, marked as answer]
Hi Martina!

Thanks for checking out oce!

The short answer is that in general, no, it is not possible to get pressure/depth from in-situ measurements of S/C and T. Further, you actually need to *know* the pressure/depth in order to convert measurements of temperature and conductivity into salinity. Of course, if you're instrument is only ever used in very shallow water, you can probably ignore the pressure effects. For example, the difference in inferred salinity between a pressure of 0 and 50 dbar is:

```
> swSCTp(1, T90fromT68(15), 0) - swSCTp(1, T90fromT68(15), 50)
[1] 0.020332
```
or, expressed as a percentage, only about 0.05%
```
> (swSCTp(1, T90fromT68(15), 0) - swSCTp(1, T90fromT68(15), 50))/35 * 100
[1] 0.05809144
```

Be careful though -- a salinity difference of 0.02 is actually about 6 times the accuracy of most research CTD systems.

The only way you could really do this is if you already had a really good relationship between T/S and depth that you could use to "infer" the depth for an individual measurement. But that supposes you've already used an instrument that *does* measure depth, and also that the relationship between T/S and depth hasn't changed (which in the ocean it almost certainly will).

Hope that helps!
Clark

**Comment by dankelley** [comment]
Hi.  I just happened across this discussion item, and see that it was initially addressed to me.  As for my own answer, it aligns with Clark's.  There is no way forward, that I can think of. 

If you have existing data, I suggest you try to look at other things, taking pressure to be an unknown.  For example, maybe there is something interesting about how temperature varies over time, or how a temperature-salinity relationship does so, etc.

If you are planning to acquire new data, I recommend you borrow or buy something to measure pressure.  I am not clear on the application.  If your data are from a mooring, you could perhaps use depth-sounder information to determine the depth at which data were acquired.  Or, if the device is lowered over the side of a vessel -- which is what I am guessing -- maybe you could try to use some information about the lowering velocity together with time, or the number of turns of a winch, or some other approach.  But, really, it's not terribly hard to get a pressure measurement.  If you have the budget to rent time on a vessel, then perhaps you can acquire a bit more funding to buy instrumentation that will serve your purposes better.
